extension/js/agent/utils/stackFrame.js

```diff
@@ -1,15 +1,15 @@
-
 ;(function(Agent) {
+
```

Comment:
Sorry, I've gotta say it. Isn't this func cool. I really like having this around